src/components/Card.jsx

```diff
@@ -0,0 +1,16 @@
+const Card = ({ article }) => {
+  return (
+    <div className="article-card">
+      <h3 className="article-title">{article.title}</h3>
+      <div className="article-meta">
+        <span className="article-author">By {article.author}</span>
+        <div className="article-stats">
+          <span className="votes">👍 {article.votes}</span>
+          <span className="comments">💬 {article.comment_count}</span>
```

Comment:
It may be nice to add the image_url in here too, just to add a bit more variety to the page.